src/renderer/assets/styles/bookDetailsDialog.css

```diff
@@ -16,7 +16,7 @@
 
 .c_dialog_background {
     position: fixed;
-    z-index: 101;
+    z-index: 102;
```

Comment:
should a similar implementation strategy be used for all popup modal dialogs? (for example, yes/no/cancel confirmations, LCP passphrase input, etc.)

Reply:
I tested a few dialogs, it all seems to work consistently.